src/rules/stylistic-issues.js

```diff
@@ -0,0 +1,135 @@
+        'max-nested-callbacks': ['error', {max: 3}],                                              //  enforce a maximum depth that callbacks can be nested
+        'max-params': ['error', {max: 5}],                                                        //  enforce a maximum number of parameters in function definitions
+        'max-statements': 'off',                                                                  //  enforce a maximum number of statements allowed in function blocks
+        'max-statements-per-line': ['error', {max: 1}],                                           //  enforce a maximum number of statements allowed per line
+        'multiline-comment-style': 'off',                                                         //  enforce a particular style for multiline comments
+        'multiline-ternary': 'off',                                                               //  enforce newlines between operands of ternary expressions
+        'new-cap': 'error',                                                                       //  require constructor names to begin with a capital letter
+        'new-parens': 'error',                                                                    //  require parentheses when invoking a constructor with no arguments
+        'newline-per-chained-call': ['error', {ignoreChainWithDepth: 2}],                         //  require a newline after each call in a method chain
+        'no-array-constructor': 'error',                                                          //  disallow Array constructors
+        'no-bitwise': 'error',                                                                    //  disallow bitwise operators
+        'no-continue': 'off',                                                                     //  disallow continue statements
+        'no-inline-comments': 'off',                                                              //  disallow inline comments after code
+        'no-lonely-if': 'error',                                                                  //  disallow if statements as the only statement in else blocks
+        'no-mixed-operators': 'error',                                                            //  disallow mixed binary operators
+        'no-mixed-spaces-and-tabs': 'error',                                                      //  disallow mixed spaces and tabs for indentation
+        'no-multi-assign': 'off',                                                                 //  disallow Use of Chained Assignment Expressions
+        'no-multiple-empty-lines': ['error', {max: 1}],                                           //  disallow multiple empty lines
+        'no-negated-condition': 'error',                                                          //  disallow negated conditions
+        'no-nested-ternary': 'error',                                                             //  disallow nested ternary expressions
```

Comment:
We seem to use (and override) this quite a bit now. Do we still want this rule to error?

Reply:
I would be super happy to turn this off and see if devs can use them responsibly >_>